Pacman remove (#14807)
tldr-pages/tldr: pages/linux/pacman-remove.md

> See also: `pacman`.
> More information: <https://manned.org/pacman.8>.

- [R]emove a package and its dependencies recur[s]ively:

`sudo pacman -Rs {{package}}`

> - [R]emove a package and both its dependencies and configuration files ([n]):

Reviewer: ```suggestion
- [R]emove a package and its dependencies. Also do [n]ot save backups of configuration files:
```

> - Remove a package and both its dependencies and configuration files ([n]):

Author: `-n` also removes the configuration files from .config/

> - Remove a package and both its dependencies and configuration files ([n]):

Reviewer: Ideally the mnemonic should be tacked after a word that's at least similar to the longform option

`sudo pacman -Rs -n {{package}}`

- Remove a package without prompting:

`sudo pacman -R --noconfirm {{package}}`

- Remove orphan packages (installed as dependencies but not required by any package):

> `sudo pacman -R -n $(pacman -Qdtq)`

Author: it should be `-Rs -n` . Corrected in the commit.

> `sudo pacman -Rs -n $(pacman -Qdtq)`

Reviewer: ```suggestion
`sudo pacman -Rsn $(pacman -Qdtq)`
```

> - [R]emove a package and all packages that depend on it:

Reviewer: ```suggestion
- [R]emove a package and [c]ascade that to all packages that depend on it:
```
